chanzuckerberg/single-cell-data-portal
# feat: cxg conversion script updates for uns

## Reason for Change

- #6659

## Changes

- Added logic to extract `uns` dictionary from AnnData and store object under `uns` directory in the `cxg`
- Cast `in_tissue` column values to boolean to make it categorical, and fix the issue of it appearing as "continuous" in explorer
- Added unit test coverage

## Testing steps

run `def make_cxg()` in a notebook or from the command line against a valid AnnData/h5ad dataset.
run converted `cxg` in explorer, verify `spatial` mode

## Notes for Reviewer
